kernels/test/op_linear_test.cpp

```diff
@@ -43,16 +44,17 @@ class OpLinearOutTest : public OperatorTest {
       }
     }
 
-    // matmul gives 32 * 2 * 3 = 192
+    // matmul gives 32 * 2 * 3 + 1 = 193
     Tensor x = tf.full({3, 32}, 2);
     Tensor y = tf.full({5, 32}, 3);
+    Tensor bias = tf.full({5}, 1);
 
     // Output shape should be (3, 5)
     Tensor out = tf.zeros({3, 5});
 
-    op_linear_out(x, y, out);
+    op_linear_out(x, y, bias, out);
 
-    Tensor expected = tf.full({3, 5}, 192);
+    Tensor expected = tf.full({3, 5}, 193);
 
     EXPECT_TENSOR_EQ(out, expected);
```

Comment:
please add bias checking as a second op_linear_out call in this function rather than replacing the existing one; as written, the empty-bias path is now untested.